Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-name: aerospike-rest-client
-description: Helm Chart For Aerospike REST Client On Kubernetes
-version: 0.1.0
+name: aerospike-rest-gateway
+description: Helm Chart For Aerospike REST Gateway On Kubernetes
+version: 1.0.0
```

Comment:
Is this the first helm chart for rest gateway? Are we okay starting with 1.0 version?

Reply:
We are in a bit of a weird spot due to the name change from "rest-client" to "rest-gateway". It is technically the same application, and I am assuming this is a breaking change.  I assume any users of the old version will also see it that way.

Reply:
Cool.

Reply:
I think it's going to get treated as a new chart (of a new software) altogether because of the rename. So, we don't really need to start as `1.0.0`. We can also start versioning from `0.1.0` based on the how ready this helm chart is. `version` field is for the helm chart, while `appVersion` indicates the rest gateway release. 

Just my thoughts. I would leave it to you and others to decide the version to start with.

Reply:
It's too early to update it to v1.